Swift 4

florieger opened the pull request:
Created a Swift 4.x package for the kata after attending the Crafting Code workshop at DevTernity.

Advantages:
- Latest Swift syntax.
- Run tests from the command line. Xcode is not required.
- Faster test execution, since the package doesn't need to start the iOS simulator.

Further details and requirements can be found in the [README.md](https://github.com/florieger/trip-service-kata/blob/swift-4/Swift%204.x/TripServiceKata/README.md).

I decided to create a new folder called `Swift 4.x`.  
The old `Swift 2.x` folder still exists.

sandromancuso commented:
Hi Alessandro. 

Thanks for your contribution. Would you mind removing the copyright (see below) from all the files? I'll be happy to merge it. 

+//  Created by Alessandro Benvenuti on 21/02/2016.
 +//  Copyright © 2016 Alessandro Benvenuti. All rights reserved.

Thanks

florieger commented:
Hi Sandro,

my name is Florian and I did not touch the copyright notice. 
It already exists on the current master branch.
I just copied the files from the `Swift 2.x` folder and adjusted them.

Anyway, I removed the copyright from the new files in the `Swift 4.x` folder, but the still exist in the `Swift 2.x` folder.

sandromancuso commented:
Hi Florian, 

Really sorry for mixing things up. :( 

Thanks for the contribution. I'm merging it now. 

Cheers
Sandro
